Capfile

```diff
@@ -0,0 +1,28 @@
+# Load DSL and set up stages
+require 'capistrano/setup'
+
+# Include default deployment tasks
+require 'capistrano/deploy'
+
+require 'capistrano/ssh_doctor'
+# Include tasks from other gems included in your Gemfile
```

Comment:
whenever is in the Gemfile, but not integrated with capistrano deployment process. Please add https://github.com/javan/whenever#capistrano-v3-integration